lib/dialects/postgres/query-generator.js

```diff
@@ -102,8 +102,10 @@ var QueryGenerator = {
     });
   },
 
-  showTablesQuery: function() {
-    return "SELECT table_name FROM information_schema.tables WHERE table_schema = 'public' AND table_type LIKE '%TABLE' AND table_name != 'spatial_ref_sys';";
+  showTablesQuery: function(options) {
+    options = options || {};
+    options.schema = options.schema || 'public';
```

Comment:
Hi!

Sorry for my bad english...

I want use sequelize-auto for generate models. sequelize-auto use `showAllTables`.

My next step (after U accept my commit): commit a patch to sequelize-auto for use schema option.

> 2016. márc. 2. dátummal, 9:52 időpontban Mick Hansen <notifications@github.com> írta:
> 
> In lib/dialects/postgres/query-generator.js <https://github.com/sequelize/sequelize/pull/5515#discussion_r54694477>:
> 
> > @@ -102,8 +102,10 @@ var QueryGenerator = {
> >      });
> >    },
> >  
> > -  showTablesQuery: function() {
> > -    return "SELECT table_name FROM information_schema.tables WHERE table_schema = 'public' AND table_type LIKE '%TABLE' AND table_name != 'spatial_ref_sys';";
> > +  showTablesQuery: function(options) {
> > +    options = options || {};
> > +    options.schema = options.schema || 'public';
> Have you taken a look at extractTableDetails?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sequelize/sequelize/pull/5515/files#r54694477>.
>